### PR TITLE
media-plugins/vdr-undelete: EAPI 8, remove old versions

### PR DESCRIPTION
**Pull request by madmartin**
media-plugins/vdr-undelete: EAPI 8
media-plugins/vdr-undelete: remove old versions

### CODE REVIEW

**Comment by dlan17** [comment]
can you rebase this PR to latest? as I saw some conflicts..

I'd personally keep few old versions, and usually give certain overlap time that new & old will co-exist, at least leave one old + one new, the benefit would be that users have the chance to fall back to old version if have problem with new one

so you'd better have good reason to remove the old? e.g the old is broken or has security issue?

File: media-plugins/vdr-undelete/vdr-undelete-2.3.0_pre1.ebuild [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit vdr-plugin-2
 
 DESCRIPTION="VDR Plugin: Recover deleted recordings of VDR"
 HOMEPAGE="http://phivdr.dyndns.org/vdr/"
 SRC_URI="http://phivdr.dyndns.org/vdr/vdr-undelete/vdr-undelete-2.3.1-pre1.tgz -> ${P}.tgz"
```
Comment (thesamesam) [inline media-plugins/vdr-undelete/vdr-undelete-2.3.0_pre1.ebuild:10]: Why is the version hardcoded here (and not corresponding to ${PV}, 2.3.1 vs 2.3.0)?

File: media-plugins/vdr-undelete/vdr-undelete-2.3.0_pre1.ebuild [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+S="${WORKDIR}/undelete-2.3.1-pre1"
```
Comment (thesamesam) [inline media-plugins/vdr-undelete/vdr-undelete-2.3.0_pre1.ebuild:11]: Please derive from ${P}.

File: media-plugins/vdr-undelete/vdr-undelete-2.3.0_pre1.ebuild [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
 
 SLOT="0"
 LICENSE="GPL-2"
```
Comment (thesamesam) [inline media-plugins/vdr-undelete/vdr-undelete-2.3.0_pre1.ebuild:14]: LICENSE should be above SLOT, per skel.ebuild.

File: media-plugins/vdr-undelete/vdr-undelete-2.3.0_pre1.ebuild (hunk 1 above)
Comment (madmartin) [reply media-plugins/vdr-undelete/vdr-undelete-2.3.0_pre1.ebuild]: Hi @thesamesam ,
I made a rebase with the current tree and found that you fixed this in October 2022. Your changes are back in here.

File: media-plugins/vdr-undelete/vdr-undelete-2.3.0_pre1.ebuild (hunk 2 above)
Comment (madmartin) [reply media-plugins/vdr-undelete/vdr-undelete-2.3.0_pre1.ebuild]: same as above

**Comment by madmartin** [comment]
this PR is far too old, close.

**Pull request closed without merging**